app/analytics.go

```diff
@@ -79,8 +79,6 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 			}()
 		}
 
-		dailyActiveChan := a.Srv.Store.User().AnalyticsActiveCount(DAY_MILLISECONDS)
```

Comment:
Here we want to have the "explicit async" approach because we want to run the queries concurrently. So you have to create a channel, and run the function inside a gorouting, you can see an example right here with the `AnalyticsTeamCount()` or you can see an example in the example pull request linked in the issue.